src/lib.rs

```diff
@@ -205,6 +242,12 @@ impl SchannelCredBuilder {
         self
     }
 
+    /// Sets the protocols enabled for sessions created from this builder.
+    pub fn enabled_protocols(mut self, enabled_protocols: &[Protocol]) -> SchannelCredBuilder {
+        self.enabled_protocols = Some(enabled_protocols.to_owned());
```

Comment:
....unless we unconditionally set both the client *and* server flags, which might be okay?